src/GraphQL/Builders/FieldBuilder.cs

```diff
@@ -338,11 +316,9 @@ protected FieldBuilder(FieldType fieldType)
         /// then their default values (if any) will be used.
         /// </summary>
         /// <param name="name">Directive name.</param>
+        [Obsolete("Please use the ApplyDirective method")]
```

Comment:
You know my preferences - I would remove these methods to not bother with them when doing further work in v8.

Reply:
We can always just make these into obsolete extension methods so as to not impede your field builder refactoring work. I really don’t like deleting field builder methods outright, as v5 it us expected that a user may have many of these calls in their codebase. Let’s consider the best way to implement the obsolete methods as you refactor the field builders